translations/monero-core_sv.ts

```diff
@@ -545,7 +545,7 @@
     <message>
         <location filename="../pages/Mining.qml" line="61"/>
         <source>Solo mining</source>
-        <translation>Solo-brytning</translation>
+        <translation>Enskild utvinning</translation>
```

Comment:
What's your objection to "solo" here? Singing by a single person is "solosång" in Swedish, solo dancing is "solodans", if you like to go sailing alone you're into "solosegling" etc.  This is the pattern used in English to derive the term "solo mining". Any reason why we couldn't do it in Swedish?

Reply:
Solo is rarely used outside showbiz. Solosegling is more of a slang term for ensamsegling. Solo is generally a very non-formal term.

Reply:
I don't agree with your suggestion "enskild" because the translation would mean something different than the source.

The key part of "enskild" is "-skild" which indicates separation and privacy (See e.g. https://svenska.se/tre/?sok=enskild). Examples: "enskilt samtal"="private conversation", "enskilt rum"="separate room". None of which concept is part of "solo mining".

Pool miners and solo miners enjoy the same level of privacy, and nothing is implied about separation. You could have 100 miners in the same rack in a data center, half of which are solo mining and the others are pool mining. Pool versus solo mining is only about the division of labor.

Showbiz does provides a pretty good analogy, actually. If you have a choir of singers and few solo singers, they all share in the greater effort of performing a concert, but there is division of labour. The choir members pool and combine their efforts whereas the solo singers don't. Just like mining where all involved work toward the greater good of securing the blockchain, pool miners having their work combined and solo miners working alone.

But the word 'solo' itself has nothing to do with showbiz. Look it up in a dictionary and you'll find that in English as well as in Swedish it's a loanword from Italian (which in turn has it from Latin "solus"). It simply means "ensam" ("alone") and may be used as an adjective, adverb or noun.

And that's how it's used. Someone who isn't a team player is a "solospelare", whether the term is used in politics or sports. "soloflygning" (solo flight), is an integral part of every pilot curriculum, and so on. And I'd say it's a stretch to call these slang terms. Anyone who sees the word "soloseglare" for the first time immediately understands it's a person who sails alone. Whereas there's no way you can understand the word "stålar", for example, just by looking at it. Someone has to tell you that it's a word used in certain groups to mean "money". That's what slang is.

To me it seems wasteful to try and find an alternative to a perfectly good pattern that already exists in Swedish. Even more so since it matches the English term so well.